versionable/.editorconfig

```diff
@@ -0,0 +1,15 @@
+# EditorConfig is awesome: https://EditorConfig.org
+
+# top-most EditorConfig file
+root = true
```

Comment:
we already have an editorconfig that's root at the root of the repo

Reply:
Woops that's an artefact from when I was experimenting, I should just remove this file